src/libraries/System.Console/tests/ReadAndWrite.cs

```diff
@@ -258,7 +258,8 @@ private static unsafe void ValidateConsoleEncoding(Encoding encoding)
         }
     }
 
-    [Fact]
+    [Fact] 
```

Comment:
nit: remove whitespace:

```suggestion
    [Fact]
```